src/shared/Core/InputArguments.cs

```diff
@@ -91,13 +92,13 @@ public Uri GetRemoteUri(bool includeUser = false)
                 return null;
             }
 
+            var ub = new UriBuilder();
+
             string[] hostParts = Host.Split(':');
             if (hostParts.Length > 0)
             {
-                var ub = new UriBuilder(Protocol, hostParts[0])
-                {
-                    Path = Path
-                };
+                ub.Scheme = Protocol;
+                ub.Host = hostParts[0];
```

Comment:
Is there a reason we can't continue to use the [`UriBuilder(string schemeName, string hostName)`](https://learn.microsoft.com/en-us/dotnet/api/system.uribuilder.-ctor?view=net-7.0#system-uribuilder-ctor(system-string-system-string)) constructor here?

```suggestion
                var ub = new UriBuilder(Protocol, hostParts[0]);
```

Reply:
Nope, just a hangover from a previous iteration. Will update!